cli/src/semgrep/commands/scan.py

```diff
@@ -298,6 +299,30 @@ def convert(
         type=OutputFormat,
         flag_value=OutputFormat.VIM,
     ),
+    # Names of this group are "outputs_<format>" so that they end up
```

Comment:
the pysemgrep group and comments on the option is mostly useless now because this is handled first by osemgrep, so semgrep scan --help will actually not run the Click code and those groups but instead will run the cmdliner man page provided by osemgrep.